addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -456,11 +460,6 @@ var FileWidget = SearchableMediaWidget.extend({
                 self.$media.css(style);
             }
 
-            if (self.options.onUpload) {
```

Comment:
Maybe move that into a single commit mentioning that is was introduced by ... and removed by ... ?

Reply:
ok